content/en/blog/2021-06-23-welcome-martin-hickey.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Martin Hickey Joins Helm Org Maintainer"
```

Comment:
> Martin Hickey Joins Helm Org Maintainer

perhaps "Martin Hickey Joins Helm Org As Maintainer"?

Reply:
or "Martin Hickey Joins Helm Org Maintainers"?

Reply:
Oops... missed the S when I copy/pasted from Hackmd. Fixed.